schematic/manifest/generator.py

```diff
@@ -83,7 +83,16 @@ def __init__(
         # Determine whether current data type is file-based
         is_file_based = False
         if self.root:
-            is_file_based = "Filename" in self.sg.get_node_dependencies(self.root)
+            try:
```

Comment:
I would move this outside this particular call, looking for filename. Perhaps just upstream after you call the schemaGenerator You could call `is_class_in_schema` (might be `sg.se.is_class_in_schema` from the schema explorer. If it evaluates to false, you could raise the error and break.

Reply:
That makes sense!